http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/JsonPayloadIT.java

```diff
@@ -74,71 +71,68 @@ public void sendLargeComplexJSONPayload() throws IOException {
-    public void sendJSONWithEmptyFieldsPayload() throws IOException {
+    public void sendJSONWithEmptyFieldsPayloadNoResourceValidation() throws IOException {
         String jsonPayload = readJSONFile("football-empty-teams.json");
-
-        getApp().given()
+        Response response = getApp().given()
                 .contentType(ContentType.JSON)
                 .body(jsonPayload)
                 .when()
                 .post("/football/upload-football-json")
                 .then()
-                .statusCode(BAD_REQUEST_STATUS_CODE)
-                .body(containsString("Some fields are empty or null"));
+                .statusCode(CREATION_STATUS_CODE).extract().response();
+        JsonPath jsonPath = response.jsonPath();
+        // Check that there are 2 teams with empty colorTShirt fields
+        assertThat(jsonPath.getList("teams.findAll { it.colorTShirt == '' }"), hasSize(2));
     }
 
     @Test
     public void sendInvalidJSONPayload() throws IOException {
         String jsonPayload = readJSONFile("football-teams_invalid.json");
-
         getApp().given()
                 .contentType(ContentType.JSON)
                 .body(jsonPayload)
                 .when()
                 .post("/football/upload-football-json")
                 .then()
-                .statusCode(BAD_REQUEST_STATUS_CODE)
+                .statusCode(INTERNAL_SERVER_ERROR_STATUS_CODE)
                 .body(containsString(
-                        "Invalid JSON - JsonProcessingException : Unexpected character ('{' (code 123)): was expecting comma to separate Array entries"));
+                        "Internal error: Invalid token=CURLYOPEN at (line no=7, column no=3, offset=81). Expected tokens are: [COMMA]"));
     }
 
     @Test
-    public void sendASingleJSONObject() throws JsonProcessingException {
+    public void sendASingleJSONObject() {
         FootballTeam teamX = new FootballTeam(
                 "Single Team",
                 "Pink",
                 3,
-                new Date(1900, 1, 1),
+                LocalDate.of(1900, 1, 1),
                 "A",
                 Arrays.asList("You", "You"));
-        ObjectMapper objectMapper = new ObjectMapper();
-        String jsonPayload = objectMapper.writeValueAsString(teamX);
 
         getApp().given()
                 .contentType(ContentType.JSON)
-                .body(jsonPayload)
+                .body(teamX)
                 .when()
                 .post("/football/upload-football-json")
                 .then()
-                .statusCode(BAD_REQUEST_STATUS_CODE)
+                .statusCode(INTERNAL_SERVER_ERROR_STATUS_CODE)
                 .body(containsString(
-                        "JsonMappingException : Cannot deserialize value of type `java.util.HashSet<io.quarkus.ts.http.advanced.reactive.FootballTeam>`"));
+                        "Incorrect position for processing type: interface java.util.List. Received event: START_OBJECT Allowed: [START_ARRAY]"));
```

Comment:
Here as well. Will this error consistent or it have part which can change in future? I know that all the log/error string can be changed but I would select the part which won't be potentially changed like `START_OBJECT` or `START_ARRAY`

Reply:
If the json file remains unchanged, the error will be exactly this.